playbook.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+
+  tasks: 
+    - shell: echo hello world
+      sudo: yes
+    - shell: curl -IL tut.by
```

Comment:
All tasks should have a distinct name for readability and for ``--start-at-task`` to work